app/views/teammate_invitation_mailer/create.md.erb

```diff
@@ -0,0 +1,13 @@
+Hi there!
+
+You've been invited to participate in a Call for Proposals for the event
```

Comment:
Let's rephrase to: 

You've been invited to the <%= @teammate.event %> team to help with the Call for Proposals as <%= @teammate.role %>.